deno/payloads/v8/application.ts

```diff
@@ -91,6 +91,23 @@ export interface APIApplication {
 	 * See https://discord.com/developers/docs/resources/application#application-object-application-flags
 	 */
 	flags: ApplicationFlags;
+	/**
+	 * Up to 5 tags describing the content and functionality of the application
+	 */
+	tags?: [string, string?, string?, string?, string?];
+	/**
+	 * Settings for the application's default in-app authorization link, if enabled
+	 */
+	install_params?: APIApplicationInstallParams;
+	/**
+	 * The application's default custom authorization link, if enabled
+	 */
+	custom_install_url?: string;
+}
+
+export interface APIApplicationInstallParams {
+	scopes: string[];
```

Comment:
Could this type be used instead?
https://github.com/discordjs/discord-api-types/blob/fa54b9de5522b9fa9d5367650950f8b0e44f6e14/payloads/v9/oauth2.ts#L5